src/aliases/api/handlers/handlers.go

```diff
@@ -125,9 +127,13 @@ func (h *AliasHandler) getAlias(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	err = infrahttp.WriteJSON(w, types.AliasResponse{
-		Value: alias.Value,
-	})
+	resp := types.AliasResponse{
+		AliasValue: types.AliasValue{
```

Comment:
It is not really nested because it is [embedded](https://golang.org/doc/effective_go#embedding).

It doesn't contain it the registry name and the key because it is stored in the URL path parameters `/registries/:registryName/aliases/:aliasKey`.